fix-engine/src-core-pp/parse_datetime.iml

```diff
@@ -40,7 +40,10 @@ let parse_time_ms str =
         (fun h m s ms ->
           let ms =
             if ms = "" then None
-            else Scanf.sscanf (CCString.pad ~side:`Right ~c:'0' 3i ms) ".%03d" (fun x -> Some (Z.of_int x))
+            else (match CCString.split_on_char '.' ms with
```

Comment:
A small improvement might be to use `CCString.chop_prefix ~pre:"." ms`. Up to you.

http://c-cube.github.io/ocaml-containers/3.12/containers/CCString/index.html#val-chop_prefix